.changes/v3.6.0/816-improvements.md

```diff
@@ -0,0 +1,4 @@
+* `resource/vcd_network_isolated_v2` add support for `metadata` so this field can be set when creating/updating isolated networks. [GH-816]
+* `datasource/vcd_network_isolated_v2` add support for `metadata` so this field can be retrieved when reading from isolated networks. [GH-816]
+* `resource/vcd_network_routed_v2` add support for `metadata` so this field can be set when creating/updating routed networks. [GH-816]
+* `datasource/vcd_network_routed_v2` add support for `metadata` so this field can be retrieved when reading from routed networks. [GH-816]
```

Comment:
Maybe we should add note that using XML API until OpenAPI support is in alpha? Or are you expecting that nothing will have to be changed in Terraform - that we'll just upgrade the govcd?